README.md

```diff
@@ -39,3 +43,34 @@ eligibility = alma_client.payments.eligibility(payment_data)
 if eligibility.eligible:
     alma_client.payments.create(payment_data)
 ```
+
+
+### Asynchroneous client
+
+
+```python
+from alma import AsyncClient
+
+alma_client = AsyncClient.with_api_key("sk_test..")
+payments = await alma_client.payments.fetch_all()
+
+for p in payments:
+    print(f"{p.id}: Paiement en {len(p.payment_plan)} fois")
+
+
+payment_data = {
+    "purchase_amount": 10000,
```

Comment:
Sorry I just updated `README.md` on `main` with a corrected version of that code, which misses the wrapping `"payment"` object. Can you rebase?